package.json

```diff
@@ -45,7 +45,9 @@
     "@eva-design/dss": "^1.0.1",
     "@eva-design/processor": "^1.0.1",
     "hoist-non-react-statics": "^3.2.1",
-    "lodash.merge": "^4.6.1"
+    "lodash.merge": "^4.6.1",
+    "react-native-eva-icons": "^1.1.0",
```

Comment:
Do we really need to provide these dependencies if the user didn’t install the icon module?

Reply:
This is a project dependencies, not a ui module